app.js

```diff
@@ -35,18 +61,16 @@ Words only consist of letters, never apostrophes or other punctuation symbols.
  */
 const mostCommonWord = (paragraph, banned) => {
   var noPunctuationParagraph = paragraph.toLowerCase().replace(/(!|;|'|,|\.|\?)/g,"").trim();
+  console.log("The paragraph written in lowercase and without punctuation is the following: " ,noPunctuationParagraph);
+
   var wordsInPargrphArr = noPunctuationParagraph.split(" "); 
- 
-  var bannedArray = []; 
-  bannedArray.push(banned); 
+  wordsInPargrphArr = wordsInPargrphArr.filter(word => word != banned);
```

Comment:
Always use !== instead of != (and same for ===) in Javascript, unless you're specifically looking to coerce values. I know the udemy course has more info somewhere in the first half